front/app/containers/Admin/projects/project/participationContext/utils/votingMethodUtils.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactNode } from 'react';
+import BudgetingInputs from '../components/voting/votingMethodInputs/BudgetingInputs';
+import { VotingInputsProps } from '../components/VotingInputs';
+
+/*
+  Configuration Descriptions:
+  - getVotingMethodInputs: renders specific voting method inputs in back office settings
+  */
+
+export type VotingMethodType = 'budgeting';
+
+export type VotingMethodConfig = {
+  getVotingMethodInputs?: (
+    props: VotingInputsProps
+  ) => ReactNode | JSX.Element | null;
+};
+
+const budgetingConfig: VotingMethodConfig = {
+  getVotingMethodInputs: (props: VotingInputsProps) => {
+    return <BudgetingInputs props={props} />;
+  },
+};
+
```

Comment:
Later on, we can just add in rankingConfig, singleVoteConfig, multipleVoteConfig etc..